spec/services/shopify_import/importers/custom_collections_importer_spec.rb

```diff
@@ -0,0 +1,20 @@
+require 'spec_helper'
+
+RSpec.describe ShopifyImport::Importers::CustomCollectionsImporter, type: :service do
+  before { authenticate_with_shopify }
+
+  describe '#import!', vcr: { cassette_name: 'shopify_import/custom_collections_importer/import' } do
+    it 'creates shopify data feeds' do
+      expect { described_class.new.import! }.to change(Shopify::DataFeed, :count).by(2)
+    end
+
+    it 'creates spree taxonomy' do
+      expect { described_class.new.import! }.to change(Spree::Taxonomy, :count).by(1)
+    end
+
+    it 'creates spree taxons' do
+      # One of taxons is taxonomy root.
```

Comment:
I think these comments are a good way to clear the confusion about something that is"common sense", while it's actually not always common sense. :)

👍